src/App.jsx

```diff
@@ -1,33 +1,59 @@
 import './App.scss';
+import { useState, useMemo } from 'react';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+export const App = () => {
+  const [searchQuery, setSearchQuery] = useState('');
+
+  const visibleMovies = useMemo(() => {
+    let result = [...moviesFromServer];
+
+    if (searchQuery.length) {
+      result = result.filter((movie) => {
+        const title = movie.title.toLowerCase();
+        const description = movie.description.toLowerCase();
+        const query = searchQuery.trim().toLowerCase();
+
+        return title.includes(query) || description.includes(query);
+      });
+    }
+
+    return result;
```

Comment:
Redudant reasign
Array.filter do not mutate origin array
```suggestion
    if (searchQuery.length) {
      return moviesFromServer.filter((movie) => {
        const title = movie.title.toLowerCase();
        const description = movie.description.toLowerCase();
        const query = searchQuery.trim().toLowerCase();

        return title.includes(query) || description.includes(query);
      });
    }

    return moviesFromServer;
```